package-lock.json

```diff
@@ -2020,7 +2020,7 @@
       "resolved": "https://registry.npmjs.org/@sindresorhus/is/-/is-0.14.0.tgz",
```

Comment:
Do you know what the cause of these changes to the package-lock is?

Reply:
I had to run `--legacy-peer-deps` with npm to get packages installing properly on v7. Given that I haven't installed anything new, I shouldn't have committed this anyways, and will remove